<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="guide_link_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>User guide:</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>811</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>20</width>
      <height>681</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="choose_eq_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose the equation</string>
    </property>
   </widget>
   <widget class="QLabel" name="f1_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>f1</string>
    </property>
   </widget>
   <widget class="QLabel" name="f2_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>f2</string>
    </property>
   </widget>
   <widget class="QLabel" name="f3_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>f3</string>
    </property>
   </widget>
   <widget class="QLabel" name="choose_method_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose the method</string>
    </property>
   </widget>
   <widget class="QLabel" name="chrods_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Chords</string>
    </property>
   </widget>
   <widget class="QLabel" name="newton_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Newton's</string>
    </property>
   </widget>
   <widget class="QLabel" name="simple_iteration_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Simple_iteration</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="eq_group_box">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>41</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="f1_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="f2_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="f3_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="method_group_box">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>250</y>
      <width>41</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="chords_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="newton_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="simple_iteration_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="segment_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter the segment</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="a_line_edit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="a_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>a</string>
    </property>
   </widget>
   <widget class="QLabel" name="b_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>370</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>b</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="b_line_edit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>370</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="choose_sys_label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose the system</string>
    </property>
   </widget>
   <widget class="QLabel" name="sys1_label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>sys1</string>
    </property>
   </widget>
   <widget class="QLabel" name="sys2_label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>150</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>sys2</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="sys_group_box">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>70</y>
      <width>41</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="sys1_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="sys2_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="approx_label_sys">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>230</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Initial approximation</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="x_approx_sys_line_edit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>260</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="pig_img">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>280</y>
      <width>291</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string>pig</string>
    </property>
   </widget>
   <widget class="QLabel" name="x_approx_label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>260</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="y_approx_label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>260</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="y_approx_sys_line_edit">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>260</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="solve_nle_button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>540</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>solve</string>
    </property>
   </widget>
   <widget class="QPushButton" name="solve_sonle_button">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>540</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>solve</string>
    </property>
   </widget>
   <widget class="QLabel" name="inaccuracy_label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>340</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Inaccuracy</string>
    </property>
   </widget>
   <widget class="QLabel" name="eps_label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>370</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>eps</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eps_line_edit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>370</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="inaccuracy_sonle_label">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>230</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Inaccuracy</string>
    </property>
   </widget>
   <widget class="QLabel" name="eps_sonle_label">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>260</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>eps</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eps_sonle_line_edit">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>260</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
